### DISCUSSION TITLE
does vrl support include  common vrl files?

### DISCUSSION THREAD

**Discussion by lddlww** [vectordotdev/vector#21917] [Q&A] [answered]
like other config software, common config in common file , and using include!(/path/to/common.vrl) for include it , so we do not need to write multiple same vrl configurations

best wishes

**Comment by pront** [comment, marked as answer]
Hi @lddlww,

The closest thing that we support is reading multiple files in the Remap transform: https://github.com/vectordotdev/vector/issues/21485

However, I think this open feature request is the closest thing to what you are asking for https://github.com/vectordotdev/vrl/issues/1020.

**Reply by lddlww** [reply]
great job，thanks
